core/management/commands/data_intake.py

```diff
@@ -11,9 +13,12 @@ class Command(BaseCommand):
     def add_arguments(self, parser: CommandParser) -> None:
         parser.add_argument("files", nargs="+", type=str)
 
-    def update_db(self, file, cols_as_expected) -> None:
-        data = pd.read_csv(file)
-        filename = file.split("\\")[-1].split(".")[0]
+    def update_db(self, file_path, cols_as_expected) -> None:
+        data = pd.read_csv(file_path)
+        filename = Path(Path(file_path).name).stem
```

Comment:
Maybe simplify this line. You don't need multiple calls to Path I think.

`Path(file_path).stem` should do the trick by itself.

Reply:
Done.
It does read better now. Thank you.